onnx/onnxifi_dummy.c

```diff
@@ -136,3 +141,25 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
 onnxReleaseGraph(onnxGraph graph) {
   return ONNXIFI_STATUS_SUCCESS;
 }
+
+/*
+ * ONNXIFI Extension Functions
+ */
+
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+onnxGetExtensionFunctionAddress(
+    onnxBackendID backendID,
+    const char* name,
+    onnxExtensionFunctionPointer* function) {
+  return ONNXIFI_STATUS_SUCCESS;
```

Comment:
could we test that actual function will be assigned when name == "onnxSetIOAndRunGraph"?

Reply:
Yeah, that's exactly I am doing... The main reason why this is "Unfinished" hahah